azure-pipelines.yml

```diff
@@ -0,0 +1,20 @@
+# Xcode
+# Build, test, and archive an Xcode workspace on macOS.
+# Add steps that install certificates, test, sign, and distribute an app, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/xcode
+jobs:
+  - job: func_test_ios1
+    pool:
+      vmImage: 'macOS-10.14'
```

Comment:
(notice) I'd like to have `parameters` section, but can't have it in `azure-pipelines.yml`.

[ref]
https://github.com/appium/ruby_lib_core/blob/master/ci-jobs/functional_test.yml